apps/src/aichat/views/AichatView.tsx

```diff
@@ -20,6 +23,23 @@ const AichatView: React.FunctionComponent = () => {
     dispatch(sendSuccessReport('aichat'));
   }, [dispatch]);
 
+  const initialAiCustomization = useAppSelector(
+    state =>
+      (state.lab.levelProperties as AichatLevelProperties | undefined)
+        ?.initialAiCustomizations || EMPTY_AI_CUSTOMIZATIONS
+  );
+  const aiCustomizations: AiCustomizations = {
```

Comment:
Yeah, looking at this, curious if you think it would be helpful to have the values and visibilities split up so you could just call dispatch with the values directly instead of mapping to this object?